apps/backend/src/modules/users/user.service.ts

```diff
@@ -1,11 +1,15 @@
+// import { AuthError, ErrorMessage, HTTPCode } from "shared";
+import { UserDto } from "shared/src/modules/users/users.js";
```

Comment:
Remove comment. Do not import directly from shared folder here. You should import from ./libs/types/types.js